test/parallel/test-net-better-error-messages-port-hostname.js

```diff
@@ -3,12 +3,12 @@ const common = require('../common');
 const net = require('net');
 const assert = require('assert');
 
-const c = net.createConnection(common.PORT, '***');
+const c = net.createConnection(0, '***');
```

Comment:
`createConnection()` can not take port 0 the way `listen()` and `bind()` can. Sending port 0 to createConnection() is an error.